main.js

```diff
@@ -0,0 +1,142 @@
+
+const submitPostButton = document.getElementById("new-post-button");
+
+
+function deletePost(event) {
```

Comment:
Suggestion: add comments to your code explaining what the function does. Recommend using JSDoc.